contracts/orderbook/Logic/OrderBook.sol

```diff
@@ -0,0 +1,413 @@
+pragma solidity ^0.8.0;
+import "../Storage/OrderBookEvents.sol";
+import "../Storage/OrderBookStorage.sol";
+import "../../swaps/ISwapsImpl.sol";
+import "../OrderVault/IDeposits.sol";
+import "../../interfaces/IDexRecords.sol";
+import "../../mixins/Flags.sol";
+contract OrderBook is OrderBookEvents, OrderBookStorage, Flags {
+    using EnumerableSet for EnumerableSet.Bytes32Set;
+
+    function initialize(address target) public onlyOwner {
+        _setTarget(this.getSwapAddress.selector, target);
+        _setTarget(this.getFeed.selector, target);
+        _setTarget(this.getDexRate.selector, target);
+        _setTarget(this.clearOrder.selector, target);
+        _setTarget(this.prelimCheck.selector, target);
+        _setTarget(this.queryRateReturn.selector, target);
+        _setTarget(this.priceCheck.selector, target);
+        _setTarget(this.executeOrder.selector, target);
+        _setTarget(this.setVaultAddress.selector, target);
+    }
+
+    function _executeTradeOpen(
+        IOrderBook.Order memory internalOrder
```

Comment:
you can just use `order` instead of `internalOrder`

Reply:
true